okio/src/commonTest/kotlin/okio/OkioMessageDigestTest.kt

```diff
@@ -0,0 +1,81 @@
+package okio
+
+import okio.ByteString.Companion.decodeHex
+import okio.internal.commonAsUtf8ToByteArray
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class OkioMessageDigestTest {
+
+  // region SHA-1
+
+  @Test fun sha1EmptyValueHashIsCorrect() {
+    // arrange act
```

Comment:
omit these comments? it feels like training wheels

Reply:
Sure thing.